circus-train-s3-s3-copier/src/main/java/com/hotels/bdp/circustrain/s3s3copier/S3S3Copier.java

```diff
@@ -292,7 +299,7 @@ private Copy submitCopyJob(CopyJobRequest copyJob) {
                   copyObjectRequest.getSourceBucketName(),
                   copyObjectRequest.getSourceKey());
           LOG
-              .debug("Copy failed with exception:", e);
+              .warn("Copy failed with exception:", e);
```

Comment:
The reason I say this is because we usually log errors on warn/error level so if we don't then it's usually been done like that for a reason. My assumption is that we have some retry logic so this was causing the logs to be noisy and was downgraded to debug but I could be wrong, in which case `warn` is fine.

Reply:
I changed this because during my testing of all of this since some of the failures we had around the testing of this made those errors fail silently. If we're running in production, we don't necessarily want to have debug logs on but we still want to see what the failure for a given job was.

Reply:
OK, well, let's leave it like this then and if it later becomes apparent why we put this down to debug we'll just just put it back again.